FlashDevelop/Utilities/SemVer.cs

```diff
@@ -0,0 +1,51 @@
+﻿namespace FlashDevelop.Utilities
```

Comment:
I believe it should be added to PluginCore and not into FlashDevelop. Plugins aren't supposed to reference the FlashDevelop project itself.